src/test/features/response/routes/check-and-send.ts

```diff
@@ -128,9 +128,7 @@ describe('Defendant response: check and send page', () => {
                   'Statement of truth',
                   'I believe that the facts stated in this response are true.' ,
                   'The hearing requirement details on this page are true to the best of my knowledge.',
-                  '<input id="signedtrue" type="checkbox" name="signed" value="true"'
-                ))
-                .expect(res => expect(res).to.be.successful.withoutText(
+                  '<input id="signedtrue" type="checkbox" name="signed" value="true"',
                   'Timeline of what happened',
```

Comment:
How are these passing? there should always be a timeline of what happened/ evidence

Reply:
This change puts them in the `.withText` block

Reply:
ah right yes! sorry, read it too quickly